wizard/WizardModeSelection.qml

```diff
@@ -57,16 +57,16 @@ Rectangle {
 
             WizardHeader {
                 title: qsTr("Mode selection.") + translationManager.emptyString
-                subtitle: qsTr("Please select the statement that best matches you.") + translationManager.emptyString
+                subtitle: qsTr("Please select the statement that best matches your desired setup.") + translationManager.emptyString
```

Comment:
Is this change really needed? I'm fine with editing the text if necessary, but keep in mind that everytime a string is changed even slightly, the matching string in all the language files will be resetted and will have to be retranslated. This for *more than 30 languages*.

Reply:
I don't really have a preference. This came from an issue #2208. I didn't create a branch initially and now my downstream master has this modded file in it. I'm happy to discard it, close the issues, and reclone to the latest upstream master, but didn't want to do that before resolving.